docs/release-playbook.md

```diff
@@ -0,0 +1,205 @@
+# Bazel Release Playbook
+
+Authors: lpino@, pcloudy@, laurentlb@
+
+Status: Work in progress
+
+Last Updated: 2018-11-09
```

Comment:
Remove this also, history is visible on GitHub

Reply:
Done.